msu/hooks/skills/skill.nut

```diff
@@ -246,110 +244,110 @@
 		}
 	}
 
-	local setFatigueCost = o.setFatigueCost;
-	o.setFatigueCost = function( _f )
+	// TODO: Should probably switch to a leaf hook in VeryLateBucket
+	// TODO: Should call the original before setting the base value
```

Comment:
Some TODOs here and the comment should at least be swapped to a `hookTree` reference